### PR TITLE
Added blog post about new Strimzi 0.4.0 release

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: _posts/2018-05-16-strimzi-0-4-0-release.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+date: 2018-04-21
+---
+
+We are really happy to announce the new Strimzi 0.4.0 release with a lot of fixes and new interesting features!
+
+<!--more-->
+
+# Enhanced configuration
+
+Until the previous release, just few parameters were exposed through the cluster config map so that the Kafka brokers and Zookeeper nodes configurations were limited. With this release, the `kafka-config` ([doc](http://strimzi.io/docs/0.4.0/#kafka_configuration_json_config)) and the `zookeeper-config` ([doc](http://strimzi.io/docs/0.4.0/#zookeeper_configuration_json_config)) fields were added in order to allow the user to define a full configuration through a JSON string.
```
Comment (scholzj) [inline _posts/2018-05-16-strimzi-0-4-0-release.md]: We should mentioned Connect here as well.

File: _posts/2018-05-16-strimzi-0-4-0-release.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+{
+  "num.partitions": 1,
+  "default.replication.factor": 3,
+  "log.retention.hours": 168,
+}
+```
+
+# Resources usage configuration
+
+It's now possible to specify the Kubernetes/OpenShift resource requests and limits in terms of CPU and memory usage related to the Pods running the cluster nodes. The `kafka-resources` and `zookeeper-resources` fields ([doc](http://strimzi.io/docs/0.4.0/#resources_json_config)) can be used to describe such information using a JSON string.
```
Comment (scholzj) [inline _posts/2018-05-16-strimzi-0-4-0-release.md]: Again this goes beyond just Kafka and Zookeeper. You can configure it for Connect as well as for TO.

File: _posts/2018-05-16-strimzi-0-4-0-release.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# New Kafka version
+
+Taking into account the improvements in the upstream Kafka project, the Strimzi provided Docker image for the Kafka broker is now updated to the latest 1.1.0 release.
+
+# Conclusion
+
+This release represents a step forward with some new interesting and powerful features other than bug fixes.
+Furthermore, because Strimzi is an evolving project, there are some backwards incompatible changes as well; for having more information about them you can refer to the release [changes log](https://github.com/strimzi/strimzi/releases/tag/0.4.0).
+
+What are you waiting for? Engage the community and help us to improve the **only one open source** Apache Kafka operator for running your cluster on Kubernetes/OpenShift!
```
Comment (scholzj) [inline _posts/2018-05-16-strimzi-0-4-0-release.md]: Are we really the only one opensource operator? I don't think so (https://github.com/krallistic/kafka-operator).

File: _posts/2018-05-16-strimzi-0-4-0-release.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Strimzi 0.4.0 released!"
+date: 2018-04-21
```
Comment (scholzj) [inline _posts/2018-05-16-strimzi-0-4-0-release.md]: Should we add also author name here?

File: _posts/2018-05-16-strimzi-0-4-0-release.md (hunk 1 above)
Comment (ppatierno) [reply _posts/2018-05-16-strimzi-0-4-0-release.md]: right!

File: _posts/2018-05-16-strimzi-0-4-0-release.md (hunk 2 above)
Comment (ppatierno) [reply _posts/2018-05-16-strimzi-0-4-0-release.md]: right!

File: _posts/2018-05-16-strimzi-0-4-0-release.md (hunk 3 above)
Comment (ppatierno) [reply _posts/2018-05-16-strimzi-0-4-0-release.md]: yeah :-(

File: _posts/2018-05-16-strimzi-0-4-0-release.md (hunk 4 above)
Comment (ppatierno) [reply _posts/2018-05-16-strimzi-0-4-0-release.md]: I have never added an author in the previous blog posts as well. I was thinking about a sort of author "Strimzi team" but it drives me to think about a closed team developing Strimzi which is not true.

**Pull request merged**